rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -205,6 +202,26 @@ http {
         default          $http_x_forwarded_host;
         ''               $this_host;
     }
+    {{ else }}
+    map 'dummy' $pass_access_scheme {
```

Comment:
Please add a comment at the start of the else section explaining why this is required